spec/generators/scenic/model/model_generator_spec.rb

```diff
@@ -32,5 +32,12 @@ module Scenic::Generators
       expect(model_definition).to contain("self.refresh")
       expect(model_definition).to have_correct_syntax
     end
+
+    it "does not create a factory definition" do
+      run_generator ["current_customer"]
+      factory_definition = file("spec/factories/current_customers.rb")
+
+      expect(factory_definition).to_not exist
```

Comment:
The thoughtbot styleguide is not to use "to not". As per: https://github.com/thoughtbot/guides/blob/master/style/testing/README.md